tests/Doctrine/Tests/Persistence/Mapping/FileDriverTest.php

```diff
@@ -101,6 +101,22 @@ public function testGetAllClassNamesBothSources()
         self::assertSame(['stdGlobal', 'stdGlobal2', 'stdClass'], $classNames);
     }
 
+    public function testGetAllClassNamesBothSourcesNoDupes()
```

Comment:
`: void`

Reply:
Sure, on just the new test function or all of them in that class?

Reply:
Just this one - we try to fix CS gradually, on modified code only, but we may improve this in one shot later on :)

Reply:
Done.

Reply:
Could it be that you added `void` to a totally different test method instead of your own?

Reply:
Not sure why I did that, fixed!